src/components/button/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FC, MouseEvent, ReactNode } from 'react';
+import { Button as ChakraButton } from '@chakra-ui/react';
+
+interface Props {
+  variant?: 'primary';
+  size?: 'md' | 'lg' | 'xl';
+  isDisabled?: boolean;
+  onClick: (event: MouseEvent<HTMLElement>) => void;
+  children: ReactNode;
+}
+
+const Button: FC<Props> = ({
+  variant = 'primary',
+  size = 'lg',
+  isDisabled = false,
+  onClick,
+  children,
+}) => {
+  return (
+    <>
+      <ChakraButton
```

Comment:
Don't we need the left and right icons to be exposed according to figma?

Reply:
We agreed that the initial version of Primary button should only contain text. 

Beside, UX is also developing a new Icon library, we'll need to wait them though